### PR TITLE
scroll to times after clicking a day, two column layout mobile

### LINKED ISSUE #451: Scroll to Time After Selecting Date on Mobile
### Is your proposal related to a problem?

Users are confused on mobile when tapping a date because it is not obvious when doing so that the times appear under the calendar on mobile.

### Describe the solution you'd like

Scroll to the top of the time area when date is clicked, maybe print the selected date above the available times.

### Describe alternatives you've considered

Make the times appear in a new screen all together that replaces the calendar screen.

### PR DESCRIPTION
**Pull request by PeerRich**
## What does this PR do?

- scroll to time on mobile after selecting a day
- add a two grid column for mobile

closes #451 

https://www.loom.com/share/20afe28d82214dc6968df72f2ff47577

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] go to mobile /pro
- [ ] test on a phone

### CODE REVIEW

File: apps/web/components/booking/AvailableTimes.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
           </span>
         </span>
       </div>
-      <div className="flex-grow overflow-y-auto md:h-[364px]">
+      <div className="grid flex-grow grid-cols-2 gap-x-2 overflow-y-auto sm:block md:h-[364px]">
```
Comment (PeerRich) [inline apps/web/components/booking/AvailableTimes.tsx:60]: two grid column for mobile

File: packages/ui/booker/DatePicker.tsx [hunk 2]
```diff
@@ -94,7 +96,13 @@ const Days = ({
           ) : (
             <DayComponent
               date={day}
-              onClick={() => props.onChange(day)}
+              onClick={() => {
+                props.onChange(day);
+                window.scrollTo({
+                  top: 360,
+                  behavior: "smooth",
```
Comment (PeerRich) [inline packages/ui/booker/DatePicker.tsx:103]: we could do something like "isMobile" to only scroll on mobile, however given the desktop view has more than 360px height it doesn't scroll.

it still runs "scrollTo" as an invisible operation but it's a very light JS API so I doubt we need to check if it's mobile or not

Comment (PeerRich) [reply packages/ui/booker/DatePicker.tsx:103]: @emrysal NIT: worth extracting into a named function?

Comment (emrysal) [reply packages/ui/booker/DatePicker.tsx:103]: For a single use this is fine, keep in mind window isn't always available - but it looks OK in this situation

**Review by emrysal — APPROVED** [review]
LGTM

**Pull request merged**